ext/RMagick/rmmain.c

```diff
@@ -1734,14 +1734,23 @@ version_constants(void)
 {
     const char *mgk_version;
     VALUE str;
+    VALUE nr;
     char long_version[1000];
+    size_t lib_version;
 
-    mgk_version = GetMagickVersion(NULL);
+    mgk_version = GetMagickVersion(&lib_version);
 
     str = rb_str_new2(mgk_version);
     rb_obj_freeze(str);
     rb_define_const(Module_Magick, "Magick_version", str);
 
+    // Hack due to mistake in the Windows distribution of ImageMagick
+    if (lib_version == 0x6910)
+        lib_version = 0x69A;
```

Comment:
I completely dislike this hack. This strongly depends ImageMagick versioning rule.